crates/spirv-std/Cargo.toml

```
@@ -11,8 +11,8 @@ repository.workspace = true
 spirv-std-types.workspace = true
 spirv-std-macros.workspace = true
 bitflags = "1.2.1"
-num-traits = { version = "0.2.14", default-features = false, features = ["libm"] }
-glam = { version = "0.22", default-features = false, features = ["libm"] }
+num-traits = { version = "0.2.15", default-features = false, features = ["libm"] }
+glam = { version = ">=0.22, <0.25", default-features = false, features = ["libm"] }
```

```suggestion
glam = { version = ">=0.22, <=0.24", default-features = false, features = ["libm"] }
```
(see other comment, I was confused wrt what Cargo did, but this should work)